examples/stack-full.yaml

```diff
@@ -1,20 +1,21 @@
+---
```

Comment:
Is there any functional change in this file? If not, can you remove it from the changeset?

Reply:
Curious how that got there